LensGalleyPlugin.php

```diff
@@ -253,7 +253,7 @@ public function _getXMLContents($request, $galley)
                 $referredPublication = Repo::publication()->get($galley->getData('publicationId'));
                 $referredArticle = Repo::submission()->get($referredPublication->getData('submissionId'));
             }
-            $fileUrl = $request->url(null, 'article', 'download', [$referredPublication->getData('urlPath') ?? $referredArticle->getId(), $galley->getBestGalleyId(), $embeddableFile->getId()]);
+            $fileUrl = $request->url(null, 'article', 'download',[$referredArticle->getData('id'),'version',$referredPublication->getId(), $galley->getBestGalleyId(), $embeddableFile->getId()]);
```

Comment:
I think the `->getData('urlPath')` should be preferred, when present -- and then the `->getId()` should be used as a fallback when it's not present.

Reply:
Thanks! I wil suggest this  alongside changes  to the review and close the PR